ntbs-service/Pages/Account/AccessDenied.cshtml

```diff
@@ -0,0 +1,8 @@
+@page

```

Comment:
If it's easy to re-test this, does hitting this page give a 403 response code?

Reply:
We should probably handle it as a separate ticket where we can tidy this page a bit with more detail..
I will create the ticket